test/e2e.integration.test.js

```diff
@@ -570,3 +574,31 @@ test('ensure referral balances are computed correctly', async t => {
   t.is(walletProbi.toString(), reportProbi)
   assertWithinBounds(t, amount, 5.00, 0.25, 'USD value for a referral should be approx $5')
 })
+
+test.after(async () => {
+  await Promise.all([
+    cleanEyeshadeDb([
```

Comment:
can we just delete all collections via `db.dropDatabase()`? seems like this list will be annoying to maintain